apps/frontend/src/libs/components/sidebar/sidebar-item/type.ts

```diff
@@ -0,0 +1,13 @@
+import { type ValueOf } from "@git-fit/shared";
+
+import { type IconName } from "~/libs/components/icon/types/types.js";
+import { type AppRoute } from "~/libs/enums/app-route.enum.js";
+
+type sidebarItemType = {
+	icon: IconName;
+	id: string;
+	text: string;
+	to?: ValueOf<typeof AppRoute>;
```

Comment:
we don't need id, we will use label as a key
```suggestion
	label: string;
	href: ValueOf<typeof AppRoute>;
```